### PR TITLE
Update `init` script to use correct Babel preset

### PR DESCRIPTION
**Pull request by grabbou**
Update Babel preset in `init` app.

### CODE REVIEW

**Comment by grabbou** [comment]
cc: @rafeca

**Comment by kelset** [comment]
cc @janicduplessis since you first mentioned this in https://github.com/facebook/react-native/issues/20327

File: local-cli/init/init.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ function generateProject(destinationRoot, newProjectName, options) {
     });
   }
   if (!options['skip-jest']) {
-    const jestDeps = `jest babel-jest babel-preset-react-native@^5 react-test-renderer@${reactVersion}`;
+    const jestDeps = `jest babel-jest metro-react-native-babel-preset react-test-renderer@${reactVersion}`;
```
Comment (rafeca) [inline local-cli/init/init.js]: Can you force to install the same `metro-react-native-babel-preset` version as the Metro version that RN is depending on?

**Comment by grabbou** [comment]
Going to close this one as we are experimenting inside `0.57-stable` anyway. Once it starts working there, I'll resubmit another PR with better, working, solution.

**Pull request closed without merging**